wgsl/extract-grammar.py

```diff
@@ -330,10 +330,6 @@ def parse(lines, i):
         $._reserved,
     ],
 
-    conflicts: $ => [
```

Comment:
Can we preserve conflicts as an empty list so that in case a person forks or a situation in future arises, the place can be easily found? Thanks!

Reply:
Done.